lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -997,6 +998,14 @@ SubExprInfo ExprEmitter::visitComb(ArraySliceOp op) {
   return {Unary, arrayPrec.signedness};
 }
 
+SubExprInfo ExprEmitter::visitComb(ArrayGetOp op) {
+  auto arrayPrec = emitSubExpr(op.input(), Symbol);
+  os << '[';
+  emitSubExpr(op.index(), LowestPrecedence);
+  os << ']';
+  return {Unary, arrayPrec.signedness};
```

Comment:
I think this should return Symbol or line 1002 should only require Unary, otherwise you'll get parens on 2d subscripts (array.get(array.get(x))

Reply:
Actually, I have a path pending on this. It adds a precedence level for `()`, `[]`, `::`, and `.` as is specified in the spec. It's pending on this since it is difficult to write a test which it fixes without some ops to access the arrays. I've published it: #504 .

Reply:
+1 thanks!